JASP-Engine/JASP/R/abtestbayesian.R

```diff
@@ -16,283 +16,191 @@
+                                 "orLessThan1Prob", "orGreaterThan1Prob", "orNotEqualTo1Prob", "numSamples", "bayesFactorOrder"))
+  abTestBayesianTable$position <- 1
 
-.fillABTable.abTest <- function(data = NULL, status, perform, options) {
-    # Fills and returns the complete main table
-    #
-    # Args:
-    #   data: list with footnotes and data rows calculated by .calcDataRegTable.basReg()
-    #   status: current status of the analysis
-    #   perform: 'run' or 'init'
-    #   options: a list of user options
-    #
-    # Return:
-    #   List with completed table; may be inserted in results as is
+  # abTestBayesianTable$addCitation("JASP Team (2018). JASP (Version 0.9.3) [Computer software].")
 
-    if (options$bayesFactorType == "BF10") {
-        bf.title <- "BF<sub>10</sub>"
-    } else if (options$bayesFactorType == "BF01") {
-        bf.title <- "BF<sub>01</sub>"
-    } else if (options$bayesFactorType == "LogBF10") {
-        bf.title <- "Log(BF<sub>10</sub>)"
-    }
+  if (options$bayesFactorType == "BF10") {
+      bf.title <- "BF<sub>10</sub>"
+  } else if (options$bayesFactorType == "BF01") {
+      bf.title <- "BF<sub>01</sub>"
+  } else if (options$bayesFactorType == "LogBF10") {
+      bf.title <- "Log(BF<sub>10</sub>)"
+  }
 
-    fields <- list(
-        list(name = "Models", type = "string"),
-        list(name = "P(M)", type = "number", format = "sf:4;dp:3"),
-        list(name = "P(M|data)", type = "number", format = "sf:4;dp:3"),
-        list(name = "BF", type = "number", format = "sf:4;dp:3", title = paste(bf.title, sep = ""))
-    )
+  abTestBayesianTable$addColumnInfo(name = "Models",    title = "Models",    type = "string")
+  abTestBayesianTable$addColumnInfo(name = "P(M)",      title = "P(M)",      type = "number", format = "sf:4;dp:3")
+  abTestBayesianTable$addColumnInfo(name = "P(M|data)", title = "P(M|data)", type = "number", format = "sf:4;dp:3")
+  abTestBayesianTable$addColumnInfo(name = "BF",        title = bf.title,    type = "number")
 
-    table <- list()
-    table[["title"]] <- "Model Comparison"
-    table[["citation"]] <- list()
-    table[["schema"]] <- list(fields = fields)
-
-    if (! is.null(data) && ! status$error) {
-        table[["data"]] <- data$rows
-        table[["footnotes"]] <- as.list(data$notes)
-    } else {
-        names <- sapply(fields, function(x) x$name)
-        table[["data"]][[1]] <- setNames(as.list(rep(".", length(names))), names)
-    }
+  jaspResults[["abTestBayesianTable"]] <- abTestBayesianTable
 
-    if (status$error) {
-        table[["error"]] <- list(errorType = "badData", errorMessage = status$errorMessage)
-    }
+  if (!ready)
+    return()
+
+  .abTestBayesianFillTableMain(abTestBayesianTable, ab_obj, options)
 
-    return(table)
+  abTestBayesianTable$addFootnote("A positive log odds ratio means that the success rate in Group 2 is higher than in Group 1.")
```

Comment:
I don't understand this footnote. What column are you referring to?

Reply:
This is a just an informative note for the AB Test table (https://github.com/jasp-stats/INTERNAL-jasp/issues/367)